crates/jxl-oxide/src/fb.rs

```diff
@@ -59,34 +63,49 @@ impl FrameBuffer {
             }
         }
 
-        Self {
-            width: outw,
-            height: outh,
-            channels,
-            buf,
-        }
+        out
     }
 
+    /// Returns the width of the frame buffer.
     #[inline]
     pub fn width(&self) -> usize {
         self.width
     }
 
+    /// Returns the height of the frame buffer.
     #[inline]
     pub fn height(&self) -> usize {
         self.height
     }
 
+    /// Returns the number of channels of the frame buffer.
     #[inline]
     pub fn channels(&self) -> usize {
         self.channels
     }
 
+    /// Returns the contents of frame buffer.
+    ///
+    /// The buffer has `width * height * channels` samples, where the index of `channels` varies
+    /// faster.
```

Comment:
What does "index of `channels` varies faster" mean, channels in buffer can be reordered?

Reply:
It means the first `channels` number of samples represent the first pixel. I couldn't find the way to explain this clearly.